src/libhirte/log/log.h

```diff
@@ -0,0 +1,146 @@
+#pragma once
+
+#include <stdarg.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <systemd/sd-journal.h>
+
+#include "libhirte/common/common.h"
+
+typedef enum LogLevel {
+        LOG_LEVEL_DEBUG,
+        LOG_LEVEL_INFO,
+        LOG_LEVEL_WARN,
+        LOG_LEVEL_ERROR,
+} LogLevel;
+
+// NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables)
+static const char *log_level_strings[] = { "DEBUG", "INFO", "WARN", "ERROR" };
+
+const char *log_level_to_string(LogLevel l) {
```

Comment:
Functions defined in headers like this should be static inline.

Reply:
Like the other functions, I moved its implementation to `log/log.c`.  Then it shouldn't be static inline, right?

Reply:
yeah